src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -189,6 +191,20 @@ public override void OnError(Exception ex)
                 TraceCompilationDiagnostics(exc.Diagnostics, LogTargets.User);
                 throw;
             }
+            catch (CompilerErrorException exc)
+            {
+                TraceOnPrimaryHost("Script compiler error", TraceLevel.Error, _compilationService.GetType().Name, exc);
```

Comment:
Sure... do you have any suggestions? Would removing script and saying `Compiler error` only help? Perhaps something like `Compiler service error`?